packages/fxa-settings/src/components/ExternalAlert/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { AlertBar } from '../AlertBar';
+import { useAccount } from '../../models';
+import { externalAlert } from '../../lib/cache';
+
+export const ExternalAlert = () => {
+  const account = useAccount();
+
+  return account.externalAlert ? (
+    <AlertBar
+      onDismiss={() => {
+        externalAlert(null);
+      }}
+    >
+      <p data-testid="alert-text">{account.externalAlert}</p>
```

Comment:
Can we call this `external-alert-text` (or `alert-external-text` if we change the name of the component)?